.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+# See https://pre-commit.com for more information
```

Comment:
What's this? Is it necessary for GE integration?

Reply:
Whoops, no, I didn't mean to commit this, it's a tool for running automatic pre-commit checks I was trying.  I'll remove it.